subprojects/testing-jvm/testing-jvm.gradle.kts

```diff
@@ -77,4 +77,6 @@ classycle {
 tasks.named<Test>("test").configure {
     exclude("org/gradle/api/internal/tasks/testing/junit/ATestClass*.*")
     exclude("org/gradle/api/internal/tasks/testing/junit/ABroken*TestClass*.*")
+    exclude("org/gradle/api/internal/tasks/testing/junit/ATestSetUpWithBrokenSetUp*.*")
```

Comment:
These test classes are used as "test data" and picked up by Vintage engine incorrectly.